docs/howtoguides/how_to_know_if_system_affected_by_a_list_of_cves.rst

```diff
@@ -0,0 +1,52 @@
+.. _pro-check-list-cves:
+
+Given a list of CVEs, how to know if you are affected by them?
+**************************************************************
+
+If you have a list of Common Vulnerabilities and Exposures (CVE) or Ubuntu Security Notifications (USN) and want to check if your Ubuntu system is affected by them, you can easily check using the pro fix API. In this guide we'll show you how to leverage the pro fix API:
+
+.. note::
+    The `pro fix` API is provided as a part of the `pro` package, which is a security tool for Ubuntu systems. If you don't have it installed, you can install it by following the instructions in the `pro` package documentation.
```

Comment:
It may not be obvious to a user where they can find the pro package documentation. I think we should either link to where you want people to look, or provide a 1-line command to install it here.

Reply:
(also, it's meant to be installed by default I think? so we maybe want to give the usual command of `sudo apt update && sudo apt install ubuntu-advantage-tools` (or is it `sudo apt install ubuntu-pro-client` now?)

Reply:
We can link the index of the Pro documents for more information about the package. But the pro package is default installed I assume.